components/earnings/commons/earnings_base.lua

```diff
@@ -203,9 +190,32 @@ function Earnings._buildConditions(conditions, year, mode)
 	return conditions
 end
 
----
--- customizable in case it has to be changed
--- (e.g. SC2 due to not having a fixed number of players per team)
+function Earnings._determineValue(placement, aliases, isPlayerQuery)
+	if isPlayerQuery then
+		return tonumber(placement.individualprizemoney)
+			or (placement.prizemoney / Earnings.divisionFactorPlayer(placement.mode))
```

Comment:
This is only legacy since there might be pages with outdated lpdb data, right?

Reply:
outdated data and misusage of TC/PPT combination (switched order and hence indiv prize not available)